5-Heroku/server/db/dbController.js

```diff
@@ -0,0 +1,46 @@
+require('dotenv').config('../../../.env')
+const mongoose = require('mongoose')
+const itemModel = require('../models/item')
+const url = process.env.MONGO_URL;
+
+module.exports.connectToDB = async function () {
+    await mongoose.connect(url, { useNewUrlParser: true })
+        .catch((err) => { throw err })
```

Comment:
same comment on the promise chanining. unless you do something in `then` or `catch` you don't need to write it.. the chain will propogate to the next `then` `catch` in the calling stack